### PR TITLE
completions: update based on generate help message

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: completions/bash/ocitools [hunk 1]
```diff
@@ -283,25 +284,31 @@ _ocitools_generate() {
...
 	      --pid
 	      --poststart
 	      --poststop
 	      --prestart
 	      --root-propagation
 	      --rootfs
-	      --seccomp-default
+	      --seccomp-allow
 	      --seccomp-arch
+	      --seccomp-default
```
Comment (mrunalp) [inline completions/bash/ocitools]: We would want completions for this since it is possible.

Comment (Mashimiao) [inline completions/bash/ocitools]: @mrunalp  Sorry, I don't get it.  You mean I should not change its order or I should remove it?

File: completions/bash/ocitools [hunk 2]
```diff
@@ -330,11 +337,6 @@ _ocitools_generate() {
 			return
 			;;
 
-		--seccomp-default)
-			__ocitools_complete_seccomp_actions
```
Comment (mrunalp) [inline completions/bash/ocitools]: I meant that removing this code is getting rid of the completions.

Comment (Mashimiao) [inline completions/bash/ocitools]: --seccomp-default is duplicated. I removed the redundant one

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**